### ISSUE TITLE
The Rename Method refactoring applied to an abstract method can be detected?

### ISSUE THREAD

**Issue by osmarleandro** [tsantalis/RefactoringMiner#123]
## Summary
The input for RefactoringMiner was osmarleandro/spring-boot@ad10a5c, which has applied a Rename Method refactoring. The refactoring was applied in getObjectName() method of EndpointObjectNameFactory class.

#### Code example
Diff between the commit and their parent is osmarleandro/spring-boot@ad10a5c.
```diff
@@ -104,7 +104,7 @@ class DefaultEndpointObjectNameFactoryTests {
        private ObjectName generateObjectName(ExposableJmxEndpoint endpoint) {
                try {
                        return new DefaultEndpointObjectNameFactory(this.properties, this.environment, this.mBeanServer,
-                                       this.contextId).getObjectName(endpoint);
+                                       this.contextId).getObjectName_RENAMED(endpoint);
                }

@@ -35,6 +35,6 @@ public interface EndpointObjectNameFactory {
         * @return the {@link ObjectName} to use for the endpoint
         * @throws MalformedObjectNameException if the object name is invalid
         */
-       ObjectName getObjectName(ExposableJmxEndpoint endpoint) throws MalformedObjectNameException;
+       ObjectName getObjectName_RENAMED(ExposableJmxEndpoint endpoint) throws MalformedObjectNameException;
 
 }

@@ -93,7 +93,7 @@ public class JmxEndpointExporter implements InitializingBean, DisposableBean, Be
        private ObjectName register(ExposableJmxEndpoint endpoint) {
                Assert.notNull(endpoint, "Endpoint must not be null");
                try {
-                       ObjectName name = this.objectNameFactory.getObjectName(endpoint);
+                       ObjectName name = this.objectNameFactory.getObjectName_RENAMED(endpoint);

@@ -121,12 +121,12 @@ class JmxEndpointExporterTests {
        void registerShouldUseObjectNameFactory() throws Exception {
                this.endpoints.add(new TestExposableJmxEndpoint(new TestJmxOperation()));
                this.exporter.afterPropertiesSet();
-               verify(this.objectNameFactory).getObjectName(any(ExposableJmxEndpoint.class));
+               verify(this.objectNameFactory).getObjectName_RENAMED(any(ExposableJmxEndpoint.class));
        }
 
        @Test
        void registerWhenObjectNameIsMalformedShouldThrowException() throws Exception {
-               given(this.objectNameFactory.getObjectName(any(ExposableJmxEndpoint.class)))
+               given(this.objectNameFactory.getObjectName_RENAMED(any(ExposableJmxEndpoint.class)))

```

### Environment details
RefactoringMiner 2.0.2

## Steps to reproduce
1. Run RefactoringMiner and pass as input the commit osmarleandro/spring-boot@ad10a5c.

### Actual results
RefactoringMiner yields and empty set of refactoring.

### Expected results
A single Rename Method refactoring from block code in getObjectName() method of EndpointObjectNameFactory class.

**Comment by tsantalis** [comment]
As explained in our paper (Section 4.5) changes in the signature of abstract methods (or interface methods) can be detected only by inference, i.e., only if similar changes take place is the signature of the methods overriding the abstract method.
In your commit, only the call sites have been updated, but there is no update in methods overriding `getObjectName()`.

Are these commits artificially created, or are they real ones? I believe if I tried to build the project from this commit, the build would fail with compilation errors.
